lib/makara/logging/subscriber.rb

```diff
@@ -2,12 +2,17 @@ module Makara
   module Logging
 
     module Subscriber
+      IGNORE_PAYLOAD_NAMES = ["SCHEMA", "EXPLAIN"]
 
       def sql(event)
         name = event.payload[:name]
-        name = [current_wrapper_name(event), name].compact.join(' ')
-        event.payload[:name] = name
-        super(event)
+        if IGNORE_PAYLOAD_NAMES.include?(name)
```

Comment:
This should just use `ActiveRecord::LogSubscriber::IGNORE_PAYLOAD_NAMES` directly if possible. Are there AR version semantics?